libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/Schemas/Microsoft.Test.Script.schema

```diff
@@ -31,6 +31,12 @@
                 "$kind": "Microsoft.Test.IUserTokenMock"
             }
         },
+        "calledAsLeafSkill": {
```

Comment:
>calledAsLeafSkill [](start = 9, length = 17)

Might it make sense to make this an enum across the possible kinds of mock cases the middleware supports?  And if you did that you could do the skill bot ids as well for optional values.